ci/install-bazel-rbe.sh

```diff
@@ -10,4 +24,5 @@ if [[ -n "$BAZEL_RBE_CREDENTIAL" ]]; then
     echo "The RBE credential has been installed!"
 else
     echo "No RBE credential found. Bazel will be executed locally without RBE support."
+    install_rpmbuild
```

Comment:
Let's put the code outside of the if else, so that it is always installed regardless of whether you've RBE.

Is there a better file to host the code? I think `install-bazel-rbe.sh` should only be for installing RBE.

Reply:
@lolski 
>it is always installed regardless of whether you've RBE.

This is redundant — RBE builds do not require `rpmbuild` installed and we'll lose ~15 seconds per each build

>Is there a better file to host the code? I think install-bazel-rbe.sh should only be for installing RBE.

`rpmbuild` is always needed, regardless of platform. In that way, I feel installing `rpmbuild` _only_ in `else` branch, when RBE is not present, is an implementation detail. 

A bigger idea for refactor might be merging install *all* needed deps for build (`bazel`, `rbe credential`, `rpmbuild`) in a script named `install-build-tools`

Reply:
> This is redundant — RBE builds do not require rpmbuild installed and we'll lose ~15 seconds per each build

True, however imagine a user running Bazel command without RBE, in a machine which has RBE credential which will cause it to fail. That would be quite a confusing error. I think it's better to be slower for 15 seconds rather than be confusing.

> A bigger idea for refactor might be merging install all needed deps for build (bazel, rbe credential, rpmbuild) in a script named install-build-tools

This sounds like a good idea in the future. Can you create a Github issue for that?

Reply:
@lolski addressed in https://github.com/graknlabs/build-tools/pull/78/commits/e80d9ee9c06241a881ac24a8426c8f46a8e24032

Reply:
@lolski issue https://github.com/graknlabs/build-tools/issues/79 created